src/lib/plugin-verify-github.js

```diff
@@ -0,0 +1,29 @@
+const parseSlug = require('parse-github-repo-url')
```

Comment:
+1 on moving this into separate module.
Also, I have forked the parse-github-repo-url some time ago, because they would not merge the pull request.
I added robust github parsing and some simple inhouse gitlab url parsing (hit same problem trying to set this up against gitlab at work). It is literally single dependency drop-in replacement from `parse-github-repo-url` to `@bahmutov/parse-github-repo-url`